crates/driver/src/infra/api/routes/solve/mod.rs

```diff
@@ -12,8 +12,14 @@ async fn route(
     state: axum::extract::State<State>,
     auction: axum::Json<dto::Auction>,
 ) -> Result<axum::Json<dto::Solution>, axum::Json<Error>> {
-    let auction = auction.0.into_domain()?;
+    let auction = auction.0.into_domain(state.eth()).await.map_err(|err| {
+        tracing::warn!(?err, "error creating auction");
+        err
+    })?;
     let competition = state.competition();
-    let (solution_id, score) = competition.solve(&auction).await?;
+    let (solution_id, score) = competition.solve(&auction).await.map_err(|err| {
+        tracing::warn!(?err, "error solving auction");
+        err
+    })?;
```

Comment:
Tap tap tap tap.

[Tap.](https://lib.rs/crates/tap)

:no_entry: No tap? :no_entry_sign: 

:tada: Tapstravaganza :man_dancing:

Reply:
![image](https://user-images.githubusercontent.com/15960117/218876472-f45600f9-d3cb-413b-9c79-ed4d2232f3be.png)

![image](https://user-images.githubusercontent.com/15960117/218877024-35cd5ed9-96fa-4441-a8ef-8e2d67eecbf4.png)

![image](https://user-images.githubusercontent.com/15960117/218877239-d27e9c66-a4c5-4620-9aee-348c34ac942a.png)

![image](https://user-images.githubusercontent.com/15960117/218877343-1181715b-df21-4c11-990e-6f59a401588c.png)

![image](https://user-images.githubusercontent.com/15960117/218877469-c066d586-4e80-44c5-b515-fa4b7493d3f1.png)

![image](https://user-images.githubusercontent.com/15960117/218877571-e30dc370-bd05-40f7-a786-b3f4560192a3.png)

![image](https://user-images.githubusercontent.com/15960117/218877739-5ba4883f-2ff8-4cb3-8fd8-b5563d168bc0.png)

Reply:
I'm aware of this. There is also `inspect_err` in the std but it's nightly. To be clear, you want to use the dependency?

Reply:
Changed it to tap.